### PR TITLE
#167313357 setup sass processing on the application

### PR DESCRIPTION
**Pull request by nshutijonathan**
#### What does this PR do?
It configures sass pre-processing on the application
#### Description of Task to be completed?
- Installed and setup sass pre-processing on the application to enables the use of both .scss and .css files on the client
- renamed a my branch very well to embrace recommended naming convention
#### How should this be manually tested?
* After cloning the repo,  checkout into ` ch-set-up-sass-client-processing-167313357 ` branch then npm install, after cd into server/styles/sass 
#### What are the relevant pivotal tracker stories?
[#167313357](https://www.pivotaltracker.com/story/show/167313357)
#### screenshots
image
image

### CODE REVIEW

**Review by habinezadalvan — APPROVED** [review]

**Review by minega25 — APPROVED** [review]
LGTM

**Review by salviosage — APPROVED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -8,7 +8,9 @@
     "build": "webpack --env.environment=production --config webpack.config.js",
     "dev": "webpack --env.environment=development --config webpack.config.js",
     "devserver": "webpack-dev-server --open --hot --env.environment=development --config webpack.config.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "sass": "node-sass --include-path scss src/styles/sass/*.scss src/styles/sass/*.css",
```
Comment (SokoPaulSokool) [inline package.json]: It would be a good idea for us to use loaders for sass

**Review by SokoPaulSokool — CHANGES_REQUESTED** [review]
Discuss with your teammates on how you are planning to structure your files for sass components and tests

File: package.json (hunk 1 above)
Comment (nshutijonathan) [reply package.json]: I have used sass loaders, thank you for the feedback

**Review by Anguandia — APPROVED** [review]
LGTM

**Review by nkalyesubula — APPROVED** [review]
LGTM

**Pull request merged**